core/common/src/main/java/alluxio/RuntimeConstants.java

```diff
@@ -0,0 +1,30 @@
+/*
```

Comment:
would it make sense to just make this part of the `Constants.java` file? 


the only place where `Constants.VERSION` is used is here so we could just not expose it (or adopt a convention that appends `_KEY` to all the property keys ... which I think would be useful anyhow)